plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinition.tsx

```diff
@@ -147,9 +180,20 @@ export const OpenApiDefinition = ({ definition }: OpenApiDefinitionProps) => {
     return () => clearTimeout(timer);
   }, [definition, setDef]);
 
+  // isolate the entity annotations that are supportedSwaggerPlugins AND are set to true
+  // then setup the values needed by the SwaggerUI
+  const appliedSwaggerPlugins = Object.keys(supportedSwaggerPlugins)
+    ?.filter(plugin => entity?.metadata?.annotations?.[plugin] === 'true')
+    ?.map(plugin => supportedSwaggerPlugins[plugin]);
```

Comment:
@Rugvip, Does this work for you rather than switching over the plugins to another object as well? I really don't think there is a need to duplicate the list of what swagger plugins are supported. So now instead of the annotations containing a list of the swagger plugins, they will each be their own annotation, and will be identified here.

Reply:
Hey @Rugvip, do you this code fits the general use case well enough?